src/backend/demo/management/commands/create_demo.py

```diff
@@ -303,6 +303,29 @@ def create_demo(stdout):  # pylint: disable=too-many-locals
 
         queue.flush()
 
+    with Timeit(stdout, "Creating mailboxes"):
+        domains_ids = list(
+            mailbox_models.MailDomain.objects.values_list("id", flat=True)
+        )
+        for domain_id in domains_ids:
+            for i in range(random.randint(1, 21)):
+                first_name = fake.first_name()
+                last_name = fake.last_name()
+                local_part = f"{first_name.lower()}.{last_name.lower()}{i}"
+                print(local_part)
+                queue.push(
+                    mailbox_models.Mailbox(
+                        first_name=first_name,
+                        last_name=last_name,
+                        local_part=local_part,
+                        domain_id=domain_id,
+                        secondary_email=f"{local_part}@example.fr",
+                        status=random.choice(MailboxStatusChoices.values),
+                    )
+                )
+
+        queue.flush()
```

Comment:
For reasons, `make demo` fails here because of supposed duplicates in `dn_emails`. By changing randint to (1,1) by example, it's easy to see that we don't have no duplicates (not even on `local_mailbox` parameter). 

@qbey could you please take a look ?